src/main/kotlin/io/kotlintest/matchers/ComparableMatchers.kt

```diff
@@ -0,0 +1,49 @@
+package io.kotlintest.matchers
+
+@Deprecated("Use `value shouldBe lt(x)` or `value should beLessThan(x)`")
```

Comment:
What is the idea behind introducing a new class which methods are born deprecated ?